packages/react/src/isomorphicClerk.ts

```diff
@@ -579,13 +577,13 @@ export class IsomorphicClerk implements IsomorphicLoadedClerk {
     }
   }
 
-  __unstable__updateProps = (props: any): any => {
-    // Handle case where accounts has clerk-react@4 installed, but clerk-js@3 is manually loaded
-    if (this.clerkjs && '__unstable__updateProps' in this.clerkjs) {
-      (this.clerkjs as any).__unstable__updateProps(props);
-    } else {
-      return undefined;
-    }
+  __unstable__updateProps = (props: any): void => {
+    void this.#waitForClerkJS().then(clerkjs => {
+      // Handle case where accounts has clerk-react@4 installed, but clerk-js@3 is manually loaded
+      if (clerkjs && '__unstable__updateProps' in clerkjs) {
+        (clerkjs as any).__unstable__updateProps(props);
+      }
+    });
```

Comment:
Aren't we the only ones that we will be using `__unstable__updateProps` ?
Is this public API ?
I propose to update this to be 
```suggestion
  __unstable__updateProps = (props: any): Promise<void> => {
    return this.#waitForClerkJS().then(clerkjs => {
      // Handle case where accounts has clerk-react@4 installed, but clerk-js@3 is manually loaded
      if (clerkjs && '__unstable__updateProps' in clerkjs) {
        (clerkjs as any).__unstable__updateProps(props);
      }
    });
```

Reply:
I was thinking exactly the same thing, but I think we have users that use this property... But, I guess it does not really change something for them, as it was `void` before.

Plus, this way we can await it correctly 🤔

Reply:
Done! ✔️